### ISSUE TITLE
Windows: Missing Fortran solvers

### ISSUE THREAD

**Issue by capobiag** [conda-forge/assimulo-feedstock#19]
Issue:
I installed Assimulo trough conda on Windows (Command: "conda install assimulo") and got a partially working package, i.e. I can access and use the Sundials solvers, but none of the others (RADAU5, DOPRI5,...). I suppose there is a problem with the Fortran solvers. They are not compiled during the installation process even though I have a working gfortran compiler from mingw-w64. 

Any ideas where the problem could be? Some patch in the recipe maybe?

Thanks,
Giuseppe

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at C:\ProgramData\Miniconda3\envs\myEnv:
#
# Name                    Version                   Build  Channel
alabaster                 0.7.12                     py_0    conda-forge
asn1crypto                0.24.0                py36_1003    conda-forge
assimulo                  2.9                 py36_vc14_6  [vc14]  conda-forge
astroid                   2.0.3                 py36_1000    conda-forge
attrs                     18.2.0                     py_0    conda-forge
babel                     2.6.0                      py_1    conda-forge
backcall                  0.1.0                      py_0    conda-forge
blas                      1.0                         mkl  
bleach                    3.0.2                      py_0    conda-forge
ca-certificates           2018.10.15           ha4d7672_0    conda-forge
certifi                   2018.10.15            py36_1000    conda-forge
cffi                      1.11.5          py36hfa6e2cd_1001    conda-forge
chardet                   3.0.4                 py36_1003    conda-forge
cloudpickle               0.6.1                      py_0    conda-forge
colorama                  0.4.0                      py_0    conda-forge
cryptography              2.3.1            py36h74b6da3_0    conda-forge
cryptography-vectors      2.3.1                 py36_1000    conda-forge
cycler                    0.10.0                     py_1    conda-forge
decorator                 4.3.0                      py_0    conda-forge
docutils                  0.14                  py36_1001    conda-forge
entrypoints               0.2.3                 py36_1002    conda-forge
freetype                  2.9.1             he8b6a0d_1004    conda-forge
icc_rt                    2017.0.4             h97af966_0  
icu                       58.2                 ha66f8fd_1  
idna                      2.7                   py36_1002    conda-forge
imagesize                 1.1.0                      py_0    conda-forge
intel-openmp              2019.1                      144  
ipykernel                 5.1.0           py36h39e3cac_1001    conda-forge
ipython                   7.1.1           py36h39e3cac_1000    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
isort                     4.3.4                 py36_1000    conda-forge
jedi                      0.13.1                py36_1000    conda-forge
jinja2                    2.10                       py_1    conda-forge
jpeg                      9b                       vc14_1  [vc14]  conda-forge
jsonschema                3.0.0a3               py36_1000    conda-forge
jupyter_client            5.2.3                      py_1    conda-forge
jupyter_core              4.4.0                      py_0    conda-forge
keyring                   16.1.0                py36_1000    conda-forge
kiwisolver                1.0.1           py36he980bc4_1002    conda-forge
lazy-object-proxy         1.3.1           py36hfa6e2cd_1000    conda-forge
libpng                    1.6.35            h7602738_1002    conda-forge
llvmlite                  0.25.0                py36_1001    conda-forge
markupsafe                1.1.0           py36hfa6e2cd_1000    conda-forge
matplotlib                3.0.2           py36h8a2030e_1001    conda-forge
matplotlib-base           3.0.2           py36h3e3dc42_1001    conda-forge
mccabe                    0.6.1                      py_1    conda-forge
mistune                   0.8.4           py36hfa6e2cd_1000    conda-forge
mkl                       2018.0.3                      1  
mkl_fft                   1.0.6                    py36_0    conda-forge
mkl_random                1.0.2                    py36_0    conda-forge
nbconvert                 5.3.1                      py_1    conda-forge
nbformat                  4.4.0                      py_1    conda-forge
numba                     0.40.0           py36hf9181ef_0  
numpy                     1.15.4           py36ha559c80_0  
numpy-base                1.15.4           py36h8128ebf_0  
numpydoc                  0.8.0                      py_1    conda-forge
openssl                   1.0.2o                   vc14_0  [vc14]  conda-forge
packaging                 18.0                       py_0    conda-forge
pandoc                    2.3.1                         0    conda-forge
pandocfilters             1.4.2                      py_1    conda-forge
parso                     0.3.1                      py_0    conda-forge
pickleshare               0.7.5                 py36_1000    conda-forge
pip                       18.1                  py36_1000    conda-forge
prompt_toolkit            2.0.7                      py_0    conda-forge
psutil                    5.4.8           py36hfa6e2cd_1000    conda-forge
pycodestyle               2.4.0                      py_1    conda-forge
pycparser                 2.19                       py_0    conda-forge
pyflakes                  2.0.0                      py_0    conda-forge
pygments                  2.2.0                      py_1    conda-forge
pylint                    2.1.1                 py36_1000    conda-forge
pyopenssl                 18.0.0                py36_1000    conda-forge
pyparsing                 2.3.0                      py_0    conda-forge
pyqt                      5.6.0            py36h764d66f_7    conda-forge
pyrsistent                0.14.6          py36hfa6e2cd_1000    conda-forge
pysocks                   1.6.8                 py36_1002    conda-forge
python                    3.6.6                he025d50_0    conda-forge
python-dateutil           2.7.5                      py_0    conda-forge
pytz                      2018.7                     py_0    conda-forge
pywin32-ctypes            0.2.0                 py36_1000    conda-forge
pyzmq                     17.0.0                   py36_3    conda-forge
qt                        5.6.2           vc14h6f8c307_12  
qtawesome                 0.5.3              pyh8a2030e_0    conda-forge
qtconsole                 4.4.3                      py_0    conda-forge
qtpy                      1.5.2              pyh8a2030e_0    conda-forge
requests                  2.20.1                py36_1000    conda-forge
rope                      0.10.7                     py_1    conda-forge
scipy                     1.1.0            py36h4f6bf74_1  
setuptools                40.6.2                   py36_0    conda-forge
sip                       4.18.1           py36h6538335_0    conda-forge
six                       1.11.0                py36_1001    conda-forge
snowballstemmer           1.2.1                      py_1    conda-forge
sphinx                    1.8.2                 py36_1000    conda-forge
sphinxcontrib-websupport  1.1.0                      py_1    conda-forge
spyder                    3.3.1                    py36_2    conda-forge
spyder-kernels            0.2.6                      py_1    conda-forge
sqlite                    3.25.3            hfa6e2cd_1000    conda-forge
sundials                  2.7.0                    vc14_5  [vc14]  conda-forge
testpath                  0.4.2                 py36_1000    conda-forge
tornado                   5.1.1           py36hfa6e2cd_1000    conda-forge
traitlets                 4.3.2                 py36_1000    conda-forge
typed-ast                 1.1.0           py36hfa6e2cd_1000    conda-forge
urllib3                   1.23                  py36_1001    conda-forge
vc                        14.1                 h0510ff6_4  
vs2015_runtime            14.15.26706          h3a45250_0  
wcwidth                   0.1.7                      py_1    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.32.3                   py36_0    conda-forge
win_inet_pton             1.0.1                 py36_1002    conda-forge
wincertstore              0.2                   py36_1002    conda-forge
wrapt                     1.10.11         py36hfa6e2cd_1001    conda-forge
zlib                      1.2.11                   vc14_0  [vc14]  conda-forge

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : myEnv
    active env location : C:\ProgramData\Miniconda3\envs\myEnv
            shell level : 2
       user config file : C:\Users\ac125669\.condarc
 populated config files : C:\Users\ac125669\.condarc
          conda version : 4.5.11
    conda-build version : not installed
         python version : 3.6.6.final.0
       base environment : C:\ProgramData\Miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/win-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/win-64
                          https://repo.anaconda.com/pkgs/pro/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\ProgramData\Miniconda3\pkgs
                          C:\Users\ac125669\AppData\Local\conda\conda\pkgs
       envs directories : C:\ProgramData\Miniconda3\envs
                          C:\Users\ac125669\AppData\Local\conda\conda\envs
                          C:\Users\ac125669\.conda\envs
               platform : win-64
             user-agent : conda/4.5.11 requests/2.20.1 CPython/3.6.6 Windows/7 Windows/6.1.7601
          administrator : False
             netrc file : None
           offline mode : False


```
</details>

**Comment by jschueller** [comment]
The compilation of these is not supported by the setup.py for windows, please ask upstream to support it.

**Comment by jschueller** [comment]
This is a valid issue, I tried in #26 without luck as I dont know to force the mingw compiler instead of msvc.

**Comment by jschueller** [comment]
Maybe @chria can help here ?

**Comment by jschueller** [comment]
tried in #30, building fortran solver succeed except for py2k (msvcr link conflict, maybe retry when we drop py27 ? numpy>=1.13
but then it wont link to sundials, so maybe rebuild a static version locally

**Comment by aravindh95babu** [comment]
Having the same issue. I installed Assimulo through conda as well and I have the MinGW FORTRAN compilers installed. Any suggestions?

**Comment by jschueller** [comment]
maybe @chria can help us here

**Comment by aravindh95babu** [comment]
The build of assimulo as obtained from `conda list` is py37hf5aa970_1007 . Please let me know if any other info would be useful.

**Comment by PechoJo** [comment]
I 've installed assimulo via conda (Anaconda, 64bit, Python 3.7). And the sundials solvers are all working fine. But the others don't, like Radau5ODE. 
It would be great if someone could share experiences or solutions.

**Comment by aravindh95babu** [comment]
I had the exact same install process and the exact same symptoms. I’ve had no luck fixing it after I posted that, unfortunately. I’m still hoping for an update from the JModelica people.

**Comment by jschueller** [comment]
fixed by #45, except for GLIMDA which is not correctly linked

**Comment by bilderbuchi** [comment]
> fixed by #45, except for GLIMDA which is not correctly linked

@jschueller has the GLIMDA problem ever been fixed? I'm trying to simulate an ME FMU with pyfmi, and getting a "Could not find GLIMDA" message. It looks like this could be it? (conda, Windows 10)

**Comment by jschueller** [comment]
fortran solvers are only available for python 3.11 or older, not 3.12

**Comment by bilderbuchi** [comment]
That is on Python 3.10.8, though. 😕 
In the meantime, I've realised that that message was only a warning, anyway, and simulation seems to have completed correctly. 🤷
